jekyll/_cci2/migrating-from-1-2-experimental-translator.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: classic-docs
+title: "Experimental Translator from 1.0 to 2.0 config"
+short-title: "Experimental Translator for CircleCI 1.0 to 2.0"
+description: "Instructions for using the experimental Translator to translate CircleCI 1.0 configuration to the 2.0 format."
+categories: [migration]
+hide: true
+order: 15
+---
+
+This document provides instructions for using an experimental Translator that generates a CircleCI 2.0 configuration based on an existing 1.0 project.
+
+* Contents
+{:toc}
+
+## Important Caveat:
+This Translator is designed to produce working configuration, but not necessarily optimal configuration. The configuration generated is commented to help you understand the inferred commands that the 1.0 build engine executed. _For production use, we strongly recommend reading and manually creating your 2.0 configuration using our [documentation on migrating from 1.0 to 2.0]({{ site.baseurl }}/2.0/migrating-from-1-2/)_.
+
+## Translator Overview
+The configuration Translator is an experimental project to help with migrating from CircleCI 1.0 to 2.0. 
+
+The approach in CircleCI 2.0 differs in a few significant ways that make an automated migration complicated:
+1. **CircleCI 1.0 has "magic" inferred commands, whereas 2.0 is wholly deterministic.** In CircleCI 1.0 our Inference Engine runs implicit commands based on the contents of your repository in many cases. For instance, unless overridden, CircleCI 1.0 would detect the presence of rspec tests and run them automatically with no intervention. In 2.0 our philosophy is to run no "magic" commands, preferring instead to have all configuration be transparent and deterministic for developers. 
```

Comment:
I would personally use the word `explicit` over `transparent` in this document, I think it better conveys the reasoning behind the verbosity of 2.0 config files.